babel/src/test/resources/sql/big-query.iq

```diff
@@ -2708,6 +2708,18 @@ SELECT TIMESTAMP(DATE "2008-12-25") AS timestamp_date;
 
 !ok
 
+# This tests the no-op TIMESTAMP and DATETIME functions.
+select TIMESTAMP(TIMESTAMP "2008-01-01 01:03:05") as ts,
```

Comment:
Maybe using `!type` would be helpful here since the aim is to show they return the same type as they are given.

Reply:
or use `!ok` and `!type` in conjunction somehow